package.json

```diff
@@ -34,9 +34,9 @@
     "@metamask/eth-sig-util": "^4.0.0",
     "@mui/icons-material": "^5.8.4",
     "@mui/material": "^5.5.0",
-    "@pushprotocol/restapi": "1.7.18",
+    "@pushprotocol/restapi": "1.7.19",
```

Comment:
https://push-protocol.github.io/push-dapp/pr-preview/pr-1531/channels

The Developer section and the channels page keeps on loading on the preview URL. It is working on other PR previews.


I see a localhost error in the console. My guess is this this is related to the restapi package. Right?

<img width="2553" alt="Screenshot 2024-05-22 at 4 01 23 PM" src="https://github.com/push-protocol/push-dapp/assets/25508878/2e6bf9ec-eb4f-42fe-80a5-f7f46fdec7bf">

Reply:
Weird, let me commit a package that has